src/components/Compare/CompareTable.tsx

```diff
@@ -295,7 +273,21 @@ const CompareTable = (props: {
               </TextButton>
             )}
           </div>
-          <MoreInfoButton onClick={onClickFAQ} />
+          <ShareImageButtons
```

Comment:
moves share buttons under compare (in place of the 'More Info' button that triggered the now unused FAQ modal)